docs/docs-ref-autogen/excel/excelscript/excelscript.rangeformat.yml

```
@@ -262,7 +323,31 @@ methods:
       content: 'getRowHeight(): number;'
       return:
         type: number
-        description: ''
+        description: |-
+
+
+          #### Examples
+
+          ```TypeScript
+          /**
+           * This script doubles the row height for every row in the active worksheet's used range.
+           */
+          function main(workbook: ExcelScript.Workbook) {
+            const currentSheet = workbook.getActiveWorksheet();
+            const usedRange = currentSheet.getUsedRange();
+
+            // Get all the current row heights before setting them to improve performance.
```

To clarify the purpose of this loop, I recommend switching the order of the phrases: 
```suggestion
            // To optimize performance, get all the current row heights before setting them.
```